Ground-Items.md

```diff
@@ -118,20 +121,47 @@ Configures the color for insane value items.
 
 Configures the start price for insane value items.
 
-### Only show loot
+### Ownership filter
 
-Only shows drops from NPCs and players.
+Options for deciding which type of loot displays.
 
-### Delay for double-tap ALT to hide
+All - All
+Takeable - Any Loot/Items you can pick up, including your (GIM)group loot
+Drops - Enemy drops for you, and your (GIM)group loot
```

Comment:
```suggestion
All - All ground items  
Takeable - All ground items you can pick up, including your (GIM)group loot  
Drops - Enemy drops for you, and your (GIM)group loot
```

The spaces are required so it's not all 1 line. The other changes in this suggestion are personal preference so feel free to ignore if you don't like it.

You can try to complicate this further by mentioning potential raids/SW/whatever bugs, but that's not worth it IMO unless we get a ton of support traffic about it in the future.